kong.conf.default

```diff
@@ -521,6 +532,17 @@
                                              # This value is only used during
                                              # migrations.
 
+#declarative_config =           # The path to the declarative configuration
+                                # file which holds the specification of all
+                                # entities (Routes, Services, Consumers, etc.)
+                                # to be used when Kong runs without a database.
+                                #
+                                # Entities are stored in the cache, so it
+                                # must have enough space to store all entities
+                                # (see the `mem_cache_size` directive),
+                                # and should be configured to never expire
+                                # (i.e. `db_cache_ttl` should be kept at 0).
```

Comment:
"it" sort of lacks context here. Maybe a better way to phrase this:
```
Entities are stored in Kong's in-memory cache, so you must
ensure that enough memory is allocated to it via the
`mem_cache_size` property. You must also ensure that items
in the cache never expire, which means that `db_cache_ttl`
should preserve its default value of 0.
```

Note that throughout our entire documentation, we refer to Kong's config properties as "properties", while using "directives" for Nginx directives only.

Reply:
That works too!

> Note that throughout our entire documentation, we refer to Kong's config properties as "properties", while using "directives" for Nginx directives only.

Funny, I _did_ look it up to see how other attributes were being called, and my first hit was seeing `log_level` (which is a `kong.conf` property) being called a "directive" so I went with that.

Reply:
Ugh... We should change it then!

Reply:
Can do. Should all the "See <nginx URL> for a description of this directive" be changed as well? The "this" there seems to refer to the property (pedantically speaking the most technically correct would be "See <nginx URL> for a description of the Nginx directive exposed by this property")

Reply:
Scratch that, these entries mention "directive of the same name" above — my bad.